vita3k/gui/src/information_bar.cpp

```diff
@@ -185,10 +208,8 @@ void init_notice_info(GuiState &gui, EmuEnvState &emuenv) {
                     if (!set_notice_info(gui, emuenv, *notice_it)) {
                         notice_it = user.second.erase(notice_it);
                         save_notice_list(emuenv);
-                    } else {
-                        notice_info_new[notice_it->time] = notice_list_new[user.first][notice_it->time];
+                    } else
```

Comment:
```suggestion
                    } else {
```
If the if has brackets, then the else should have brackets too